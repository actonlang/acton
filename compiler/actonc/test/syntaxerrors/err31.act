# Invalid operator combination
def bad_op():
    return 5 ** ** 2