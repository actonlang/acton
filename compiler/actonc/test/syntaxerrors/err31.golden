Building file test/syntaxerrors/err31.act using temporary scratch directory
[error Parse error]: unexpected "** 2<newline>"
                     expecting atomic expression or unary operator

     +--> test/syntaxerrors/err31.act@3:17-3:18
     |
   3 |     return 5 ** ** 2
     :                 ^ 
     :                 `- unexpected "** 2<newline>"
     :                    expecting atomic expression or unary operator
     :                    
-----+
